app/handlers/ErrorHandler.scala

```diff
@@ -26,17 +26,18 @@ import views.html.{ErrorTemplate, not_found}
 import javax.inject.{Inject, Singleton}
 
 @Singleton
-class ErrorHandler @Inject()(
-                              val messagesApi: MessagesApi,
-                              view: ErrorTemplate,
-                              notFound: not_found,
-                            )(implicit appConfig: FrontendAppConfig) extends FrontendErrorHandler with I18nSupport {
+class ErrorHandler @Inject()(val messagesApi: MessagesApi,
+                             view: ErrorTemplate,
+                             notFound: not_found
+                            )(implicit appConfig: FrontendAppConfig)
+  extends FrontendErrorHandler with I18nSupport {
```

Comment:
It was okay before. Better to leave as it was before as it's not making it as long line